src/utils/ExportHelper.ts

```diff
@@ -0,0 +1,89 @@
+// Utility methods to call import/export Jahia API
+
+// http://localhost:8080/cms/export/default/export.zip?exportformat=site&live=true&users=true&path=/groups&path=/settings/mail-server-1&sitebox=digitall&exportPath=abc
+// Export
+
+import RequestOptions = Cypress.RequestOptions;
+import {JahiaServer} from '../support';
+
+const API_NAME = '/cms/export';
+
+export type ExportParameters = {
+    viewContent?: boolean;
+    viewVersion?: boolean;
+    viewAcl?: boolean;
+    viewLinks?: boolean;
+    viewMetadata?: boolean;
+    viewWorkflow?: boolean;
+    exportPath?: string;
+    exportformat: 'all' | 'site' | 'xml' | 'zip';
+    root?: string;
+    live?: boolean;
+    users?: boolean;
+    sitebox?: string[];
+    paths?: string[];
```

Comment:
The parameter seems to be named `path` (https://github.com/Jahia/jahia-private/blob/b443cf943a4376736738aeb2069017d617225022/core/src/main/java/org/jahia/bin/Export.java#L430)